### PR TITLE
when current phase is full next milestone is from next phase

### PR DESCRIPTION
**Pull request by jadmnsaad**
https://app.zenhub.com/workspaces/epictrack-63891ea941d309001fa292cf/issues/gh/bcgov/epic.track/1512

Details;
- find index of next workphase
- Get next milestone from it if current phase is complete

### CODE REVIEW

File: epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx [hunk 1]
```diff
@@ -1,121 +1,129 @@
...
+          <ETParagraph color={Palette.neutral.dark}>
+            {work?.anticipated_decision_date ?? "-"}
+          </ETParagraph>
+        </Grid>
+        <Grid item xs={6}>
+          <ETCaption1 bold color={Palette.primary.main}>
+            CURRENT MILESTONE
+          </ETCaption1>
+        </Grid>
+        <Grid item xs={6}>
```
Comment (dinesh-aot) [inline epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx:54]: Can't figure out the change. Can you mentioned what exactly changed here.

File: epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx [hunk 2]
```diff
@@ -1,121 +1,129 @@
...
+import { WorkplanContext } from "../../WorkPlanContext";
+import { ETCaption1, ETCaption2, ETParagraph, GrayBox } from "../../../shared";
+import { Palette } from "../../../../styles/theme";
+import dayjs from "dayjs";
+import { MONTH_DAY_YEAR } from "../../../../constants/application-constant";
+
+const WorkDetails = () => {
+  const { work, workPhases } = useContext(WorkplanContext);
+
+  const currentWorkPhaseIndex = workPhases?.findIndex(
```
Comment (jadmnsaad) [inline epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx:12]: use findIndex

File: epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx [hunk 3]
```diff
@@ -1,121 +1,129 @@
...
+    (phase) => phase.work_phase.id === work?.current_work_phase_id
+  );
+  const currentWorkPhase = workPhases?.[currentWorkPhaseIndex];
+  const nextWorkPhase =
```
Comment (jadmnsaad) [inline epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx:16]: get the nextWorkPhase

File: epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx [hunk 4]
```diff
@@ -1,121 +1,129 @@
...
+        <Grid item xs={6}>
+          <ETParagraph color={Palette.neutral.dark}>
+            {currentWorkPhase?.current_milestone ?? "-"}
+          </ETParagraph>
+        </Grid>
+
+        <Grid item xs={6}>
+          <ETParagraph color={Palette.neutral.dark}>
+            {currentWorkPhase?.next_milestone ??
+              nextWorkPhase?.next_milestone ??
```
Comment (jadmnsaad) [inline epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx:69]: use nextWorkPhase here

File: epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx (hunk 1 above)
Comment (jadmnsaad) [reply epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx:54]: Sorry Dinesh, added the comments

**Review by dinesh-aot — APPROVED** [review]

**Pull request merged**